.gitignore

```diff
@@ -66,6 +66,7 @@ Session.vim
 /.idea/
 /.vscode/
 /cmd/*/.ethereum/
+*.iml
```

Comment:
@cammellos I've added my IDE file to the gitignore in this pull. Any objection?

Reply:
no objections :)